src/vanilla/GeneratorSettingsTS.cs

```diff
@@ -58,17 +58,26 @@ public class GeneratorSettingsTS : IGeneratorSettings
         public bool Multiapi { get; set; }
 
         /// <summary>
-        /// The path to the "default" API version, e.g. 2016-04-01.
-        /// The presence of this property indicates that we are generating
-        /// the multi-api "root" artifacts, and not any TS source files.
+        /// The NPM package name of the "default" API version package.
+        /// e.g. "@azure/arm-storage-2018-03-01-preview"
+        /// This package will be used as a dependency in the "alias" package.
+        /// e.g. "@azure/arm-storage"
         /// </summary>
-        public string DefaultApiVersion { get; set; }
+        public string DefaultApiVersionPackage { get; set; }
```

Comment:
If you want to name this "alias", then I'd recommend changing the variable so that "alias" is in the name, such as `AliasPackageAPIVersion`.

Reply:
`bool MultiapiLatest`